javers-core/src/main/java/org/javers/core/diff/changetype/container/ArrayChange.java

```diff
@@ -13,4 +14,28 @@
     public ArrayChange(GlobalId affectedCdoId, Property property, List<ContainerElementChange> changes) {
         super(affectedCdoId, property, changes);
     }
+
+    public Object[] getLeftArray() {
```

Comment:
hmmm, ja rozumiałem ten ficzer tak, że mamy dać userowi oryginalną tablicę (starą i nową) a nie listę zmian w formie tablicy. W opisie issue nie jest to jasne, ale ja tak to rozumiem, można dopytać tomazic89...